### ISSUE TITLE
app-root-path is using "module" node module which cause issue when using at client side

### ISSUE THREAD

**Issue by dilipkumar2k6** [inxilpro/node-app-root-path#13]
I am using Angularjs 1.x and commonjs and compiling using webpack. Since my compiled js file serve on browser therefore it has to be as small as possible.
I thought to use app-root-path module for my client side as well however i realized that it throws error in webpack compilation because "resolve.js" is doing require("module"). And when I installed "module" node package then it included big files and also throws error in browser console. 

I am not expecting to solve this issue. However wanted to share that due to this issue i can't use it for client side module bundling.

**Comment by inxilpro** [comment]
Yep, this module is not meant to work on the client side.

**Comment by rudeluv** [comment]
I'm also using Browserify mostly just to pack modules for a CLI tool I'm working on. 

I found a workaround after digging through an [issue](https://github.com/electron/electron/issues/3755) and a [blame](https://github.com/electron/electron/commit/eac2e7cc614d4e98e2828921f63d55f7b5e2c85c#diff-499c5d84d3bc39887b58b121d0ce034aL19). 

Here's how I was able to get it to compile, although I think I'll just fork it and remove the electron support.

1. ``` npm install --save-dev electron-prebuilt```
2.. Modify ``` resolve.js``` remote require to ``` var remote = require('electron-prebuilt').remote; ```

After doing this I could successfully pack my build. Hope it helps.
